### PR TITLE
fixed "else" and added action for no choice

### PR DESCRIPTION
**Pull request by gungazoo**
Fixed the "else" that was failing for some reason. Also added removing compression and deduplication.

### CODE REVIEW

File: Microsoft.AVS.Management/Microsoft.AVS.Management.psm1 [hunk 1]
```diff
@@ -1655,3 +1655,66 @@ function Set-ToolsRepo
...
+        [Parameter(Mandatory = $false,
+        HelpMessage = "Enable compression and deduplication.")]
+        [bool]$Deduplication = $false,
+        [Parameter(Mandatory = $false,
+        HelpMessage = "Enable compression only.")]
+        [bool]$Compression = $false
+    )
+
+    # $cluster is an array of cluster names or "all"
+    If ($ClustersToChange -eq "*")
```
Comment (et1975) [inline Microsoft.AVS.Management/Microsoft.AVS.Management.psm1:1686]: With the wildcards there's no need to if/else branching - if user passes "*" the logic in else branch will handle it just fine.

Comment (gungazoo) [reply Microsoft.AVS.Management/Microsoft.AVS.Management.psm1]: Good point. Changed

File: Microsoft.AVS.Management/Microsoft.AVS.Management.psm1 [hunk 2]
```diff
@@ -1655,3 +1655,60 @@ function Set-ToolsRepo
...
+    foreach ($cluster_each in ($ClustersToChange.split(",",[System.StringSplitOptions]::RemoveEmptyEntries)).Trim())
+    {
+        $Clusters += Get-Cluster -Name $cluster_each
+    }
+
+    foreach ($Cluster in $Clusters)
+    {
+        $cluster_name = $Cluster.Name
+
+        If ($Deduplication -eq $true)
```
Comment (et1975) [inline Microsoft.AVS.Management/Microsoft.AVS.Management.psm1:1695]: You can probably drop `-eq $true` as `$Deduplication` is already a boolean expression.

**Review by et1975 — APPROVED** [review]

File: Microsoft.AVS.Management/Microsoft.AVS.Management.psm1 (hunk 2 above)
Comment (gungazoo) [reply Microsoft.AVS.Management/Microsoft.AVS.Management.psm1]: I removed the default values and the "-eq $true"

**Review by et1975 — APPROVED** [review]

**Pull request merged**